dhis-2/dhis-services/dhis-service-reporting/src/test/java/org/hisp/dhis/interpretation/InterpretationServiceTest.java

```diff
@@ -219,6 +229,31 @@ public void testAddComment()
         assertEquals( 2, interpretationA.getComments().size() );
     }
 
+    @Test
+    public void testMentions()
```

Comment:
Modify test to check usernames with non-alpha chars.

Reply:
Just added, another test for a user with non-alpha chars in username